File: src/242.valid-anagram.ts
// Given two strings s and t, return true if t is an anagram of s, and false otherwise.

// An Anagram is a word or phrase formed by rearranging the letters of a different word or phrase, typically using all the original letters exactly once.

function isAnagram(s: string, t: string): boolean {
  // O(n log n)
  if (s.split("").sort().toString() === t.split("").sort().toString())
    return true;
  else return false;
}

File: src/python/242.valid-anagram.py
# Given two strings s and t, return true if t is an anagram of s, and false otherwise.
# An Anagram is a word or phrase formed by rearranging the letters of a different word or phrase, typically using all the original letters exactly once.


def isAnagram(s: str, t: str) -> bool:
    # Time: O(n log n)
    # Space: O(n)
    return sorted(s) == sorted(t)
